### ISSUE TITLE
Maintenance

### ISSUE THREAD

**Issue by callbacknull** [oferei/json-gate#39]
@oferei Hey just wondering if you're still maintaining for this module? I've forked this repo under my company's git server and we utilize it as part of our API framework. In there I've added few minor features(casting values, default values from FN, Zipcode format matching for US/CA/UK) If you don't have the time in life and don't have plans on maintenance I'd be more than happy to take over maintenance otherwise I can open up PRs for some of these.

Thanks!

**Comment by oferei** [comment]
I don't have time to maintain this, plus I haven't worked on node.js for quite a while. I'd be happy to pass this project on to someone. It's a useful tool and a lot of people are using it.

**Comment by callbacknull** [comment]
Got it - I suspected as much. It happens and I totally get it

Well consider this my volunteering to take over the project! I've been working in Node for the past 4 years and currently consulting for node at a previous company I worked for. I haven't had an open-source project to manage yet but I have built & maintained a set of 7 libraries as part of an framework for building an API server in Nodejs. So I do have quite a bit of experience with package / release management.

I've worked closely with json-gate on this API framework project; it serves two critical roles in the framework. Every module I built uses the library for validating configurations and filling in default values during startup. During runtime, my framework utilizes the library for validating inputs that API endpoints are called with against defined schemas.

The API framework is now powering my company's outbound call automation; we provide communication services for the healthcare industry. I will also be pushing pretty hard for them to open source the work so that I can refactor the project into a usable portable lightweight API framework.

I originally chose json-gate for it's lightweight footprint, simplicity, and efficiency; all values I plan to adhere to with the future of json-gate
